### PR TITLE
adapt loss to full mask

### PR DESCRIPTION
**Pull request by marouaneamz**
in the pull request #1229 , for the extreme case that we have a full-mask we do for the loss = 0 which will cause problems to calculate the gradient of the loss. here i  use the torch vector of the image to create a loss =0

### CODE REVIEW

File: mmcls/models/heads/multi_task_head.py [hunk 1]
```diff
@@ -23,7 +23,8 @@ def wrapped(inputs, data_samples, **kwargs):
                 task_data_samples.append(data_sample.get(task_name))
 
         if len(task_data_samples) == 0:
-            return {'loss': torch.tensor(0.), 'mask_size': torch.tensor(0.)}
+            loss = (inputs[0] * 0).sum()
```
Comment (piercus) [inline mmcls/models/heads/multi_task_head.py:26]: @marouaneamz i think you should explain with a comment here

**Comment by Ezra-Yu** [comment]
closed since https://github.com/open-mmlab/mmpretrain/pull/1530 merged.
